cookiecutter/cli.py

```diff
@@ -67,4 +79,9 @@ def main(template, no_input, checkout, verbose):
             level=logging.INFO
         )
 
-    cookiecutter(template, checkout, no_input)
+    if extra_context:
+        # Convert tuple -- e.g.: (u'program_name=foobar', u'startsecs=66')
+        # to dict -- e.g.: {'program_name': 'foobar', 'startsecs': '66'}
+        extra_context = dict(s.split('=', 1) for s in extra_context)
+
+    cookiecutter(template, checkout, no_input, extra_context)
```

Comment:
@hackebrot: I think perhaps you are confusing the `cookiecutter` and `main` functions? `cookiecutter` is a function in another module.